op-program/host/common/common.go

```diff
@@ -110,11 +110,12 @@ func FaultProofProgram(ctx context.Context, logger log.Logger, cfg *config.Confi
 		logger.Debug("Client program completed successfully")
 		return nil
 	} else {
-		runFlag := cl.RunProgramFlagValidate
+		var clientCfg cl.Config
```

Comment:
On fork mode (`cfg.ExecCmd` is set), we need to set the `OP_PROGRAM_CLIENT_USE_INTEROP` envar) in the `cmd`. Because the op-program client binary here is a plain-ol native binary, rather than a cannon loader prestate.